server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -236,17 +238,17 @@ public void deleteConfirmedCandidates(SortedMap<String,String> confirmedDeletes)
       System.out.println("SAFEMODE: There are " + confirmedDeletes.size()
           + " data file candidates marked for deletion in " + metadataLocation + ".\n"
           + "          Examine the log files to identify them.\n");
-      log.info("SAFEMODE: Listing all data file candidates for deletion");
+      fileLog.info("SAFEMODE: Listing all data file candidates for deletion");
       for (String s : confirmedDeletes.values()) {
-        log.info("SAFEMODE: {}", s);
+        fileLog.info("SAFEMODE: {}", s);
       }
-      log.info("SAFEMODE: End candidates for deletion");
+      fileLog.info("SAFEMODE: End candidates for deletion");
       return;
     }
 
     List<String> processedDeletes = Collections.synchronizedList(new ArrayList<>());
 
-    minimizeDeletes(confirmedDeletes, processedDeletes, fs, log);
+    minimizeDeletes(confirmedDeletes, processedDeletes, fs, fileLog);
```

Comment:
Is there a benefit to allowing multiple loggers to be supplied to `minimizeDeletes` maybe a list or varargs?  During troubleshooting / debugging having the logging in `minimizeDeletes` along with the "normal" log messages might help?  The fileLog seems to be for longer term archiving - but there may be a benefit from having the log messages in the general log too.

Reply:
There's only a single log message in minimizeDeletes which needs capturing.
https://github.com/apache/accumulo/blob/b1b2557f949e9212a1b1ca9b65f2d66c01a69edb/server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java#L440

I think adding in a list of loggers is unnecessary as it complicates the codebase while a user can accomplish the same thing by enabling `additivity` to their `.files` logger definition in their log4j config file

Reply:
This is OBE as I've removed the second logger